docs/deployment.rst

```diff
@@ -68,29 +68,25 @@ Our recommended GC tuning looks like this:
 Important Configuration Options
 -------------------------------
 
-The following logistical configurations should be changed for production environments. These options depend on your cluster
-layout.
+The following logistical configurations should be changed for production environments. These options depend on your cluster layout.
```

Comment:
"Logistical"? Is this a term of art, or can we just say, "configurations"?

Reply:
Agree, removed